chart/values.yaml

```diff
@@ -14,6 +14,15 @@ timescaledb-single:
     enabled: false
   # number or TimescaleDB pods to spawn (default is 3, 1 for no HA)
   replicaCount: 1
+  persistentVolumes:
+    data:
+      size: 50Gi
+  patroni:
+    bootstrap:
+      dcs:
+        postgresql:
+          parameters:
+            max_wal_size: 5120
```

Comment:
I don't think this is the right setting. I think what we want to set is `persistentVolumes.wal.size` let's set that to 20Gi actually